lib/MessageDecoder.ts

```diff
@@ -19,7 +19,7 @@ export class MessageDecoder {
     this.registerPlugin(new Plugins.Label_10_POS(this));
     this.registerPlugin(new Plugins.Label_10_Slash(this));
     this.registerPlugin(new Plugins.Label_12_N_Space(this));
-    this.registerPlugin(new Plugins.Label_14_Slash(this));
+    this.registerPlugin(new Plugins.Label_13_18_Slash(this));
```

Comment:
is there a better way to name this? as is it sort of implies label 12 with preamble 18/. maybe its best to have an individual decoder for each label as a stub that calls an internal helper decoder?

Reply:
I could do it that way - I do with H1. Trying to see if i like this way better (i think i do). I'll think about a better name and if i can't i'll go that route